c7n/resources/mq.py

```diff
@@ -157,3 +157,26 @@ class MarkForOpMessageBroker(TagDelayedAction):
                     op: delete
                     days: 7
     """
+
+
+@resources.register('message-config')
+class MessageConfig(QueryResourceManager):
+
+    class resource_type(TypeInfo):
+        service = 'mq'
+        enum_spec = ('list_configurations', 'Configurations', None)
+        cfn_type = 'AWS::AmazonMQ::Configuration'
+        id = 'Id'
+        arn = 'Arn'
+        name = 'Name'
+        universal_taggable = object()
+
+    def augment(self, resources):
+        client = local_session(self.session_factory).client('mq')
```

Comment:
why not just fetch the metrics with universal_augment, ala augment = universal_augment its more efficient then fetching per resource as it can do batches of 20.

Reply:
Yep, that makes way more sense. I was getting an error in universal_augment because I forgot to set the arn_type, but that's fixed now.